packages/experiments/src/components/Microfeedback/Microfeedback.state.ts

```diff
@@ -0,0 +1,6 @@
+import { IMicrofeedbackViewProps, VoteType } from './Microfeedback.types';
+
+export interface IMicrofeedbackState extends IMicrofeedbackViewProps {
```

Comment:
Can move this to `Microfeedback.base` and remove this file.

Reply:
Fixed.